From 496c308fda6c6123fbc1ff743589064c70acd324 Mon Sep 17 00:00:00 2001
From: yuyz <yuyz@rock-chips.com>
Date: Fri, 28 Jul 2017 09:40:36 +0800
Subject: [PATCH] libiep: add CMakeLists.txt

Signed-off-by: yuyz <yuyz@rock-chips.com>
---
 CMakeLists.txt | 13 +++++++++++++
 1 file changed, 13 insertions(+)
 create mode 100644 CMakeLists.txt

diff --git a/CMakeLists.txt b/CMakeLists.txt
new file mode 100644
index 0000000..a37c9fe
--- /dev/null
+++ b/CMakeLists.txt
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(iep)
+
+set(LIBIEP_SRC_FILES
+	iep_api.cpp
+)
+
+add_library(iep SHARED ${LIBIEP_SRC_FILES})
+
+#install(TARGETS iep DESTINATION lib)
+install(TARGETS iep DESTINATION lib)
+install(FILES ${iep_SOURCE_DIR}/iep.h DESTINATION include/iep)
+install(FILES ${iep_SOURCE_DIR}/iep_api.h DESTINATION include/iep)
-- 
1.9.1

